docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md

```diff
@@ -3,124 +3,121 @@ title: How to create and edit Expectations based on domain knowledge, without in
 ---
 
 import Prerequisites from '../../guides/connecting_to_your_data/components/prerequisites.jsx'
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-This guide shows how to create an Expectation Suite without a sample Batch.
+This guide shows how to create an <TechnicalTag tag="expectation_suite" text="Expectation Suite" /> without a sample <TechnicalTag tag="batch" text="Batch" />.
 
-Here are some reasons why you may wish to do this but please contact us if there are other use cases we have not yet considered:
+Here are some of the reasons why you may wish to do this:
 
 1. You don't have a sample.
-
 2. You don't currently have access to the data to make a sample.
-
-3. You know exactly how you want your Expectations to be configured.
-
+3. You know exactly how you want your <TechnicalTag tag="expectation" text="Expectations" /> to be configured.
 4. You want to create Expectations parametrically (you can also do this in interactive mode).
-
 5. You don't want to spend the time to validate against a sample.
 
+If you have a use case we have not considered, please [contact us on Slack](https://greatexpectations.io/slack).
 
 <Prerequisites>
 
 - Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
```

Comment:
I wonder if adding qualifiers would help here: "A configured and defined Data Context"

Reply:
I'm uncertain what the benefit of those qualifiers would be.  We're already specifying that they need to have configured a Data Context, and the next line specifies a specific way in which it needs to be configured.  How do you see "defined" differing from/adding to "configured" in this scenario?